### ISSUE TITLE
[Join Request]

### ISSUE THREAD

**Issue by r0sshk** [BSData/boltaction#83]
Found an issue in the Armies of China. Big Sword Squad and German-Trained Nationalist Squad don't become visible for nationalist forces, because it checks vs parent, rather than vs Force. Have it fixed locally, let me in and I'll fix it on the main thing.

**Comment by amis92** [comment]
For such a small change I'd advise a PR.

**Comment by r0sshk** [comment]
I have no idea how to get that to work. So here's my file. It should have everything working, I created a Warlords, Communist and Nationalist Army and didn't find anything wrong with the lists anymore. Aside from Guerillas giving the same weird error message they gave when I first opened the original, untouched file, and I have no idea how to fix that one.
[BA  Armies of China.zip](https://github.com/BSData/boltaction/files/3199686/BA.Armies.of.China.zip)
